tests/islandora_rest.test

```diff
@@ -396,7 +396,18 @@ class IslandoraRestTestCase extends IslandoraHttpRestTestCase {
                  ISLANDORA_REST_RELATIONSHIP_GET_PERM,
                  ISLANDORA_REST_RELATIONSHIP_POST_PERM,
                  ISLANDORA_REST_RELATIONSHIP_DELETE_PERM,
-               ));
+                 ISLANDORA_VIEW_OBJECTS,
+                 ISLANDORA_MANAGE_PROPERTIES,
+                 ISLANDORA_INGEST,
+                 ISLANDORA_PURGE,
+                 ISLANDORA_VIEW_OBJECTS,
+                 ISLANDORA_METADATA_EDIT,
+                 ISLANDORA_ADD_DS,
+                 ISLANDORA_PURGE,
+                 ISLANDORA_VIEW_OBJECTS,
+                 ISLANDORA_METADATA_EDIT,
+                 ISLANDORA_PURGE
```

Comment:
Not 100% sure if it's the reason, but I suspect the duplicates here are causing the [tests to fail](https://travis-ci.org/discoverygarden/islandora_rest/jobs/174328085#L412).